_posts/course-materials/earth-analytics/week-10/in-class/2016-12-06-api03-rcurl-gapminder.Rmd

```diff
@@ -143,6 +143,8 @@ ggplot(summary_life_exp,
 We can also create a more advanced plot - overlaying the data points on top of
 our box plot. See the <a href="http://docs.ggplot2.org" target="_blank"> ggplot documentation</a> to learn more advanced `ggplot()` plotting approaches.
 
+
+<!-- Looks like the outliers are plotted twice here -->
```

Comment:
hey @mbjoseph -- i stole this code from Carson. Can you please clarify where you think it is plotted twice to help me adjust?